src/toil/wdl/wdl_functions.py

```diff
@@ -235,6 +265,68 @@ def process_infile(f, fileStore):
         raise RuntimeError('Error processing file: '.format(str(f)))
 
 
+def parse_value_from_type(in_data: Any,
+                          var_type: str,
+                          read_in_file: bool = False,
+                          file_store: Optional[AbstractFileStore] = None,
+                          **kwargs):
+    """
+    Calls at runtime. This function parses and validates input from its type. File
+    import is also handled.
+
+    For values set in a task block, set `read_in_file` to True to process and read
+    all encountered files. This requires `cwd`, `temp_dir`, and `docker` to be
```

Comment:
What are the semantics of these arguments that are required if `read_in_file` is set? They should probably be their own keyword arguments, rather than just being in `**kwargs`.